Dockerfile

```
@@ -0,0 +1,11 @@
+ARG PYTHON_VERSION=3.12
+ARG BASE_IMAGE=ghcr.io/astral-sh/uv:python${PYTHON_VERSION}-alpine
+
+FROM ${BASE_IMAGE} as base
```

Docker warns on this case mismatch:

```suggestion
FROM ${BASE_IMAGE} AS base
```